src/RunProcessAsTask.Tests/RunProcessAsTask.Tests.csproj

```diff
@@ -1,62 +1,27 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{DD13BD57-1B05-4298-A0F0-EA5729D17989}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>RunProcessAsTask.Tests</RootNamespace>
-    <AssemblyName>RunProcessAsTask.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>

```

Comment:
@MaxCheetham Then I'd recommend removing this option, and removing all `<Compile ...` items.

the default incidentally is a compile of `**\*.cs`, so if you want to *exclude* files for some reason, you can do: `<Compile Remove="path\not\to\compile.cs">`.  Although I'd personally avoid that if at all possible; it's pretty confusing to have cs files in your source directory that aren't part of the compilation.